devel.module

```diff
@@ -1737,9 +1737,11 @@ function ddebug_backtrace($return = FALSE, $pop = 0) {
       array_shift($backtrace);
     }
     $counter = count($backtrace);
```

Comment:
Unlike the other `count()`'s, this one seems to have remain untouched...?

Reply:
@BWPanda I believe that the issue happens within conditionals, when trying to count something "uncountable". If you look at the code above, you'll see that `$backtrace` is actually set to `debug_backtrace()`, which according to PHP docs always returns an array (which is "countable"). So I believe we're save here.

...besides, I would not want us to deviate too much from the 7.x commit here: https://git.drupalcode.org/project/devel/commit/bd4061f

Reply:
...if we are worried about every instance of `count()`, perhaps we should then introduce a `backdrop_count()` wrapper function, which will be providing an additional check for `instanceof Countable`. But that would be a separate issue, against core.